endtoend/evaluators/data.go

```diff
@@ -0,0 +1,37 @@
+package evaluators
+
+import (
+	"context"
+	"errors"
+
+	eth "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/endtoend/types"
+	"google.golang.org/grpc"
+)
+
+// ColdStateCheckpoint checks data from the database using cold state storage.
+var ColdStateCheckpoint = types.Evaluator{
+	Name: "cold_state_assignments_for_epoch_5_from_epoch_%d",
```

Comment:
For epoch 5? You can also use `fmt.Sprintf()` here to use the magic number var for the evaluator name.

Reply:
Oops, yeah, i'll use the constant from your suggestions